module2/lessons/index.md

```diff
@@ -17,10 +17,13 @@ title: Module 2 - Lessons
 * [Networking](https://docs.google.com/presentation/d/e/2PACX-1vQa672IV-XwWG9q-ujEJ0w72QCBzf4jVMNI-trwLuKt9kk_ewe0l6Mk6YSWoo9UYMETbJ5RZ3akKyux/pub?start=false&loop=false&delayms=3000)
 * [Getting Job Hunt Ready](https://docs.google.com/presentation/d/e/2PACX-1vTDlRAiXiR_PSsAFGQtP8je_pcUWmLdk1kYc4jO4hTyzxDuksNWUMdUITMNXp1pRFMM0gKDiAVyXL0c/pub?start=false&loop=false&delayms=3000)
 
-## Active Record
+## Databases
+* [Intro to Databases and ORMs](./databases_and_orms)
 
+## Active Record
```

Comment:
Note that when we merge this, I've just called the data section "Databases" rather than ActiveRecord and we can put any AR/SQL lesson in that section I think?

Reply:
Nevermind I just resolved this conflict